Dockerfile

```diff
@@ -35,17 +35,14 @@ RUN npm run build
 #install server dep
 WORKDIR ${BASEDIR}/${EDUMEET}/server
 
-RUN apt-get install -y git build-essential python openssl libssl-dev pkg-config 
-
-RUN npm install
-RUN npm install logstash-client
+RUN npm install;npm run build;npm install logstash-client
```

Comment:
I think maybe it could be spread this one lint to separated tasks. But it is not strong opinion.

Reply:
And replace `npm install` with `npm install --production=false` to install always develop dependencies this overrides the NODE_ENV setting

Reply:
Separated in three tasks. Only difference is that docker creates separate layer for each command.  
`npm install` replaced with `npm install --production=false`